### PR TITLE
Add WalletsInfo API page

### PR DESCRIPTION
**Pull request by petitnau**
Let me know if the interface is fine. If so, I'll expand the doc.

### CODE REVIEW

**Comment by petitnau** [comment]
Actually, forgot to inherit the multiple-wallet-input requirement. 

Edit: updated.

File: docs/api/algorand/wallet-info.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+This API endpoint is not yet implemented
+:::
+
+Returns the wallet balance and other information.
+
+## Request Parameters
+
+|     Name    |                Type               | Description |
+| ----------- | --------------------------------- | ----------- |
+| `addresses` | `string[]`                        | List of addresses of which informations will be retrieved. |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/wallet-info.mdx]: Provide the maximum number of addresses you can feed to this endpoint

File: docs/api/algorand/wallet-info.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# WalletInfo
+
+:::warning
+This API endpoint is not yet implemented
+:::
+
+Returns the wallet balance and other information.
```
Comment (nikitaeverywhere) [inline docs/api/algorand/wallet-info.mdx]: * for a list of provided wallet addresses

File: docs/api/algorand/wallet-info.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# WalletInfo
```
Comment (nikitaeverywhere) [inline docs/api/algorand/wallet-info.mdx]: I suggest we call it WalletsInfo since we work with an array of addresses.

**Review by nikitaeverywhere — APPROVED** [review]

**Review by nikitaeverywhere — APPROVED** [review]

**Pull request merged**